python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -626,6 +627,24 @@ def __set__(self, obj, _val):
         GangaObject.__setattr__(raw_obj, attr_name, final_val)
 
 
+def proxy_wrap(f):
+    # type: (Callable) -> Callable
+    """
+    A decorator to strip the proxy from all incoming arguments
+    (including ``self`` if it's a method) and add one to the return
+    value.
+    """
+
+    @functools.wraps(f)
+    def proxy_wrapped(*args, **kwargs):
+        s_args = [stripProxy(a) for a in args]
```

Comment:
No, and it needn't be since only the outer-most object in a hierarchy is proxy-wrapped.